gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
`gradle-wrapper.jar` and `gradlew` and `gradlew.bat` bat on root should be updated. Can you please rerun `gradle wrapper --gradle-version=8.8` task on your local and commit the changes?